src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -296,4 +313,90 @@ private void startSimThread() {
         });
         m_simNotifier.startPeriodic(kSimLoopPeriod);
     }
+
+
+
+    private void configureAutoBuilder() {
+        try {
+            var config = RobotConfig.fromGUISettings();
+            AutoBuilder.configure(
+                () -> getState().Pose,   // Supplier of current robot pose
+                this::resetPose,         // Consumer for seeding pose against auto
+                () -> getState().Speeds, // Supplier of current robot speeds
+                // Consumer of ChassisSpeeds and feedforwards to drive the robot
+                (speeds, feedforwards) -> setControl(
+                    m_pathApplyRobotSpeeds.withSpeeds(speeds)
+                        .withWheelForceFeedforwardsX(feedforwards.robotRelativeForcesXNewtons())
+                        .withWheelForceFeedforwardsY(feedforwards.robotRelativeForcesYNewtons())
+                ),
+                new PPHolonomicDriveController(
+                    // PID constants for translation
+                    new PIDConstants(10, 0, 0),
+                    // PID constants for rotation
+                    new PIDConstants(7, 0, 0)
+                ),
+                config,
+                // Assume the path needs to be flipped for Red vs Blue, this is normally the case
+                () -> DriverStation.getAlliance().orElse(Alliance.Blue) == Alliance.Red,
```

Comment:
Also, we shouldn't be flipping the path on our autos even though this field is radially symmetrical. It's because every field is built within a certain margin of error and the difference between the left and right side of the field changes even over the course of a match; think of how 2910 would slam into the source last year and how much that pushed it back over thee course of an event. 

For ref, see what we did in 2022 with our offset constants https://github.com/FRCTeam360/RainMakerXXII/blob/dev/src/main/java/frc/robot/commands/autos/TerminalRight/T_R_5Ball.java#L67